quay-enterprise/custom-cert-kube-apps.md

```diff
@@ -0,0 +1,40 @@
+## Using Helm Registry Plug-in with a self-signed certificate
+
+
+
+This document assumes you have deployed Quay Enterprise with a self-signed certificate.
+
+[Appr](https://github.com/app-registry/appr) makes use of the python requests library, which by default only trusts a standard set of certificates. This prevents helm from interacting with a Quay Enterprise instance that is using a self-signed certificate.
```

Comment:
... uses a [Python Requests library][python-requests], which trusts only a standard set of certificates by default. This prevents Helm from ...